File: FrontEnd/xego_rider/lib/models/Dto/login_request_dto.dart
class LoginRequestDto {
  const LoginRequestDto(
      {required this.phoneNumber,
      required this.password,
      this.fromApp = "RIDER"});

  final String phoneNumber;
  final String password;
  final String fromApp;

  factory LoginRequestDto.fromJson(Map<String, dynamic> json) {
    return LoginRequestDto(
      phoneNumber: json['phoneNumber'],
      password: json['password'],
      fromApp: json['fromApp'] ?? "RIDER",
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'phoneNumber': this.phoneNumber,
      'password': this.password,
      'fromApp': this.fromApp,
    };
  }
}

File: BackEnd/XeGo-Microservices/Services/Auth/XeGo.Services.Auth.API/Models/Dto/LoginRequestDto.cs
﻿namespace XeGo.Services.Auth.API.Models.Dto
{
    public class LoginRequestDto
    {
        public string PhoneNumber { get; set; } = "";
        public string Password { get; set; } = "";
        public string FromApp { get; set; } = "";
    }
}
